app/views/assessor_interface/further_information_requests/edit.html.erb

```diff
@@ -7,6 +7,11 @@
   <h2 class="govuk-heading-m"><%= item.fetch(:heading) %></h2>
   <%= govuk_inset_text(text: item.fetch(:description)) %>
   <%= render(CheckYourAnswersSummary::Component.new(**item.fetch(:check_your_answers))) %>
+  <% if (item[:work_history_summary_list_rows]).present? %>
```

Comment:
```suggestion
  <% if (rows = item[:work_history_summary_list_rows]).present? %>
```

This is a neat thing that lets us assign at the same time as checking for presence, and then further below we don't need to fetch the variable again:

```
<%= govuk_summary_list(rows:) %>
```